tests/time_tests/time-testhelper/main.cpp

```diff
@@ -0,0 +1,55 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "cli.h"
+#include "statistics_writer.h"
```

Comment:
@asomsiko should we move the file to `time-testhelper` folder?

Reply:
No, since that is not a public API of time-testhelper